library/sl

```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: 5288d401c76c12025e0495265067ad6a912b59ce
+GitCommit: 755652f449d6cde8cf7e36fde2e9c849ac512754
```

Comment:
Shouldn't this be `098732d7f64270125a178194dbc0847bd99c1215` (https://github.com/scientificlinux/sl-docker/commit/098732d7f64270125a178194dbc0847bd99c1215, from https://github.com/scientificlinux/sl-docker/tree/6)?

Reply:
.... yeah, looks like the automation choked again.....